### ISSUE TITLE
Kernel panic when updating to Mojave

### ISSUE THREAD

**Issue by Hi1307** [daliansky/XiaoMi-Pro#68]
When booting into the Mojave installer, the progress bar shows then it immediately kernel panics. Is there a fix to the problem?

https://pastebin.com/QE4fCvNZ

**Comment by stevezhengshiqi** [comment]
Please check the EFI folder in U-disk or the SSD is completely replaced.(Delete and copy!). I suggest that you can provide trouble shooting file in tonymacx86 and wait for professional answer.

**Comment by Hi1307** [comment]
Yes I did delete and copy it over, however I replaced VoodooI2C and VoodooI2CHID with an older version because single click on the trackpad wasn't working with the latest revision.
